presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroupManager.java

```diff
@@ -88,7 +88,7 @@ public final class InternalResourceGroupManager<C>
     private static final File RESOURCE_GROUPS_CONFIGURATION = new File("etc/resource-groups.properties");
     private static final String CONFIGURATION_MANAGER_PROPERTY_NAME = "resource-groups.configuration-manager";
 
-    private final ScheduledExecutorService refreshExecutor = newScheduledThreadPool(2, daemonThreadsNamed("ResourceGroupManager"));
+    private final ScheduledExecutorService refreshExecutor = newScheduledThreadPool(2, daemonThreadsNamed("resource-group-manager-refresher-%d-2"));
```

Comment:
Is having the "2" in the name particularly helpful? If so, can you extract the threadpool size to a constant variable? Otherwise someone might change the thread pool size without changing the thread name, and it will no longer be meaningful.